tests/Stubs/Client/SearchClientStub.php

```diff
@@ -0,0 +1,26 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\LoyaltyCorp\Auditing\Stubs\Client;
+
+use LoyaltyCorp\Auditing\Client\SearchClient;
+
+/**
+ * @coversNothing
+ */
+class SearchClientStub extends SearchClient
```

Comment:
Why are we extending the actual implementation, when we control the interface?

Reply:
Removed, remanent of earlier attempt to create abstract search client